app/v2/notifications/post_notifications.py

```diff
@@ -136,7 +137,7 @@ def post_notification(notification_type):
             reply_to_text=reply_to
         )
     else:
-        notification = process_sms_or_email_notification(
+        notification, personalisation = process_sms_or_email_notification(
```

Comment:
i'm not a huge fan of returning tuples from functions - is there any problem with setting `template_with_content.values = notification.personalisation` further down? the transformed personalisation should be populated onto the notification object right?

Reply:
Yep, good point, haven't thought of using notification attribute